lib/money/core_extensions.rb

```diff
@@ -12,7 +12,11 @@ def to_money
 #   '100'.to_money => #<Money @cents=10000>
 #   '100.37'.to_money => #<Money @cents=10037>
 class String
-  def to_money
-    empty? ? Money.empty : Money.parse(self)
+  def to_money(currency = Money.default_currency)
+    empty? ? Money.empty : Money.parse(self, currency)
+  end
+
+  def to_currency
```

Comment:
I'm not too keen on monkey patching core classes and IMO `1.to_money('EUR')` is becoming dubious if it's meant as a convenience method. For `"EUR".to_currency` I don't see the added value.